src/overlay.js

```diff
@@ -346,29 +386,80 @@
         },
 
         /**
-         * Changes the location and placement of the overlay.
+         * Changes the overlay settings.
```

Comment:
It looks like github has some issue positioning this comment in the diff.
Is it about the `update` method?
If so, do you think the following comment is not enough?

>  If an object is specified, the options are the same than the constructor except for the element which can not be changed.